Makefile

```
@@ -40,10 +40,15 @@ lint: # Run `clippy` and `rustfmt`.
 	# clippy for bls with supranational feature
 	cargo clippy --package ream-bls --all-targets --features "supranational" --no-deps -- --deny warnings
 
+	# cargo sort
+	cargo sort -g
+
 .PHONY: test
 test: # Run all tests.
 	cargo test --workspace -- --nocapture
 
 pr:
 	make lint && \
 	make test
+  # cargo sort
+	cargo sort -g
```

```suggestion
pr:
	make lint && \
	make test
```

cargo sort -g, is already included under `make lint`